include/vara/Feature/FeatureModelTransaction.h

```diff
@@ -980,6 +980,26 @@ class FeatureModelModifyTransactionBase {
 void addFeature(FeatureModel &FM, std::unique_ptr<Feature> NewFeature,
                 Feature *Parent = nullptr);
 
+/// Adds multiple Features to the FeatureModel
+///
+/// The vector of Features contains tuples of a parent and a new Feature.
+/// If a Parent is passed it needs to be already in the FeatureModel,
+/// otherwise, root is assumed as the parent Feature.
+///
+/// \param FM
+/// \param NewFeatures
+void addFeatures(
+    FeatureModel &FM,
+    std::vector<std::tuple<std::unique_ptr<Feature>, Feature *>> NewFeatures);
```

Comment:
We only have two elements, so we could just use `std::pair` here, which is a special case of tuple.